frontend/tests/controllers/TeamGroupsTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * TeamGroupsTest
+ *
+ * @author juan.pablo
+ */
+
+class TeamGroupsTest extends \OmegaUp\Test\ControllerTestCase {
+    /**
+     * Basic create team group test
+     */
+    public function testCreateTeamGroup() {
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser();
+        $name = \OmegaUp\Test\Utils::createRandomString();
+        $description = \OmegaUp\Test\Utils::createRandomString();
+        $alias = \OmegaUp\Test\Utils::createRandomString();
+
+        $login = self::login($identity);
+        $response = \OmegaUp\Controllers\Group::apiCreateTeamGroup(
+            new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+                'name' => $name,
+                'alias' => $alias,
+                'description' => $description
+            ])
+        );
+
+        $this->assertEquals('ok', $response['status']);
```

Comment:
```suggestion
        \OmegaUp\Controllers\Group::apiCreateTeamGroup(
            new \OmegaUp\Request([
                'auth_token' => $login->auth_token,
                'name' => $name,
                'alias' => $alias,
                'description' => $description
            ])
        );
```

si lo único que se asevera es que la llamada fue exitosa, para eso basta con que no arroje excepciones.

Reply:
Listo